js/ui/popup.js

```diff
@@ -601,12 +604,45 @@ var Popup = Overlay.inherit({
             this._$bottom && this._$bottom.remove();
             var $bottom = $("<div>").addClass(POPUP_BOTTOM_CLASS).insertAfter(this.$content());
             this._$bottom = this._renderTemplateByType("bottomTemplate", items, $bottom).addClass(POPUP_BOTTOM_CLASS);
+            this.$bottomToolbarContentItems = this._$bottom.find(".dx-toolbar-items-container > div");
             this._toggleClasses();
         } else {
             this._$bottom && this._$bottom.detach();
         }
     },
 
+    _getBottomToolbarWidth: function() {
```

Comment:
Can we use `this._$bottom.outerWidth()` instead of loop? There is the same behavior is in the `dxTabs._allowScrolling` method. It will good to get rid of code duplication?